js/repeater.js

```diff
@@ -370,8 +455,8 @@
 				});
 				footer.hide();
 			} else {
-				cont = this.infiniteScrollingCont;
-				data = cont.data();
+				var cont = this.infiniteScrollingCont;
+				var data = cont.data();
```

Comment:
Yeah, I totally understand hoisting and understand the mindset behind declaring all variables at the point where they would be hoisted to anyways. The purpose for this change (and many others like it throughout this refactor) was keeping in line with "10.4 Scope" of Steve McConnell's **Code Complete**, I'm keeping the "live time" as short as possible (Live Time is the number of lines of code between which a variable is first and last referenced in the code). To summarize his arguments on pages 246/247, "A short live time reduces the chance of incorrectly or inadvertently altering a variable between the places in which you intend to alter it, gives you an accurate picture of your code's intention, reduces the chance of initialization errors, and makes your code more readable"